src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -9,6 +9,7 @@
 #include "uiaTextRange.hpp"
 #include "../types/ScreenInfoUiaProviderBase.h"
 #include "../../../buffer/out/textBuffer.hpp"
+#include "../types/UiaTracing.h"

```

Comment:
i don't expect that the tests need the tracing header

Reply:
`Log::Comment(UiaTracing::convertAttributeId(uiaAttributeId).c_str());`

`UiaTracing` already has a way to convert the attribute ID into a string. Figured I'd reuse that code and display it here.